src/lib/operators/validate.cpp

```diff
@@ -74,6 +86,23 @@ std::shared_ptr<const Table> Validate::_on_execute(std::shared_ptr<TransactionCo
   auto job_end_chunk_id = ChunkID{0};
   auto job_row_count = uint32_t{0};
 
+  // In some cases, we can identify a chunk as being entirely visible for the current transaction. Simply said,
+  // if the youngest row in a chunk is visible, all other rows are older and hence visible, too. This
+  // (1) all rows in the chunk have been commited (i.e., their begin_cid has been set),
```

Comment:
(0) the chunk is immutable, i.e., no new rows can be added while this transaction is being executed